modules/ROOT/pages/_partials/sync/server-creating-a-datasource-schema-mapping.adoc

```diff
@@ -0,0 +1,137 @@
+[id="server-creating-a-datasource-schema-mapping-{context}"]
+= Creating a data source schema mapping
+
+This section describes how to create a schema in the Sync Server for the following data source mapping:
+
+[.Employee]
+|===
+| employee_id, `ID`
+| employee_name, `String`
+|===
+
+[.Category]
+|===
+| category_id, `ID`
+| employee_type, `String`
+| security_clearance, `ID`
+|===
+
+.Prerequisites
+
+* xref:getting-started-with-hello-world-{context}[].
+
+.Procedure
+
+. Edit the `index.js` and replace its contents with the following content:
++
+[source,javascript]
+----
+const express = require('express')
+const { VoyagerServer, gql } = require('@aerogear/voyager-server')
+
+// Stub that reflects data source mapping
+const Employee = [{
```

Comment:
given that this is a list of employees, probably better rename it to `Employees`.